src/tic_tac_toe_pair/ai.clj

```diff
@@ -0,0 +1,79 @@
+(ns tic-tac-toe-pair.ai
+  (:require [tic-tac-toe-pair.rules :refer [get-winning-token is-game-over?]]))
+
+(def win-score 100)
+(def loose-score -100)
```

Comment:
This should be 'lose' instead of 'loose'

Reply:
Thanks for catching this @pumpkincouture